pkg/apis/apps/v1beta1/zz_generated.conversion.go

```diff
@@ -291,7 +290,8 @@ func autoConvert_v1beta1_DeploymentSpec_To_extensions_DeploymentSpec(in *v1beta1
 		return err
 	}
 	out.Selector = (*meta_v1.LabelSelector)(unsafe.Pointer(in.Selector))
-	if err := api_v1.Convert_v1_PodTemplateSpec_To_api_PodTemplateSpec(&in.Template, &out.Template, s); err != nil {
+	// TODO: Inefficient conversion - can we improve it?
```

Comment:
@liggitt any ideas on this? Should we use `Convert` as the generated code suggests or the existing `Convert_v1_PodTemplateSpec_To_api_PodTemplateSpec`? If we prefer `Convert_v1_PodTemplateSpec_To_api_PodTemplateSpec`, maybe we should modify the generator code `staging/src/k8s.io/code-generator/cmd/conversion-gen/generators/conversion.go` first.